libs/core/src/lib/utils/services/themes.service.ts

```diff
@@ -50,7 +50,19 @@ export class ThemesService {
         },
         {
             id: 'sap_horizon',
-            name: 'Horizon'
+            name: 'Morning Horizon'
+        },
+        {
+            id: 'sap_horizon_dark',
+            name: 'Evening Horizon'
+        },
+        {
+            id: 'sap_horizon_hcb',
+            name: 'High Contrast Black Horizon'
+        },
+        {
+            id: 'sap_horizon_hcw',
```

Comment:
I'm not seeing anything change when I select any of the new themes other than "morning horizon"?